.editorconfig

```diff
@@ -10,14 +10,6 @@ indent_size = 4
 dotnet_diagnostic.CS1591.severity = none
 dotnet_sort_system_directives_first = true
 
```

Comment:
Will be gone after stylecop branch to Geir Marius. Needed to do this to get my intellisence to work..